ccm_web/client/src/components/BulkSectionCreateValidators.ts

```diff
@@ -35,29 +36,22 @@ class DuplicateSectionInFileSectionRowsValidator implements SectionRowsValidator
   }
 }
 
-class EmptySectionNameValidator implements SectionRowsValidator {
+class SectionNameLengthValidator implements SectionRowsValidator {
   validate = (sectionNames: string[]): SectionsRowInvalidation[] => {
     const invalidations: SectionsRowInvalidation[] = []
     sectionNames.forEach((sectionName, row) => {
-      if (sectionName.trim().length === 0) {
-        invalidations.push({ message: 'Empty section name is not allowed', rowNumber: row + 2, type: InvalidationType.Error })
-      }
-    })
-    return invalidations
-  }
-}
-
-class SectionNameTooLongValidator implements SectionRowsValidator {
-  validate = (sectionNames: string[]): SectionsRowInvalidation[] => {
-    const invalidations: SectionsRowInvalidation[] = []
-    sectionNames.forEach((sectionName, row) => {
-      if (sectionName.trim().length > 255) {
-        invalidations.push({ message: 'Section name must be 255 characters or less', rowNumber: row + 2, type: InvalidationType.Error })
+      const result = validateString(sectionName, sectionNameSchema)
+      if (!result.isValid) {
+        invalidations.push({
+          message: result.messages.length > 0 ? result.messages[0] : 'The section name is invalid.',
```

Comment:
You'll see a similar pattern to this in a few places. I don't think it's possible that `messages` will be of length 0, but I added this just for thoroughness.